### PR TITLE
code styling, uniformify, message updates

### PR DESCRIPTION
**Pull request by eddiemonge**
most of the requires were only used in one place so i made it like the requires in the rest of the code that only had one use.

some code styling cleanup

some message changes.

I can rework (or take out parts) that don't look right.

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -96,27 +87,33 @@ function runYo(env) {
...
   packageName: pkg.name,
   packageVersion: pkg.version
 });
 
-if (opts.insight === false) {
-  insight.config.set('optOut', true);
-} else if (opts.insight) {
-  insight.config.set('optOut', false);
-}
+insight.config.set('optOut', opts.insight === false);
```
Comment (sindresorhus) [inline lib/cli.js]: this is incorrect. please read the code carefully

**Comment by sindresorhus** [comment]
Inlining requires is a bad idea as that won't with ES6, so better to get into the habit of having them at the top.

As for the code style cleanup I don't really see any improvement there.

File: lib/cli.js [hunk 2]
```diff
@@ -24,20 +14,17 @@ var cmd = args[0];
 
 function pre() {
   if (opts.version) {
-    console.log(pkg.version);
-    return;
+    return console.log(pkg.version);
```
Comment (sindresorhus) [inline lib/cli.js]: return on a separate line makes the code more clear as it show the intent better

File: lib/cli.js [hunk 3]
```diff
@@ -60,13 +47,17 @@ function init() {
...
 
     // If no generator is passed, then start the Yo UI
     if (!cmd) {
       if (opts.help) {
-        return console.log(fs.readFileSync(path.join(__dirname, 'usage.txt'), 'utf8'));
+        return console.log(
+          require('fs').readFileSync(
+            require('path').join(__dirname, 'usage.txt'),  'utf8'
+          )
+        );
```
Comment (sindresorhus) [inline lib/cli.js]: indents like this IMHO makes it harder to read

Comment (gaboesquivel) [inline lib/cli.js]: +1 inline looks better

File: lib/cli.js (hunk 1 above)
Comment (eddiemonge) [inline lib/cli.js]: you're right. I completely glossed over the else if part. But what else would `opts.insight` be, `undefined || 0`?

File: lib/cli.js (hunk 2 above)
Comment (eddiemonge) [inline lib/cli.js]: then all the returns should be on there own line like https://github.com/yeoman/yo/blob/master/lib/cli.js#L59 and https://github.com/yeoman/yo/blob/master/lib/cli.js#L66

File: lib/cli.js (hunk 3 above)
Comment (eddiemonge) [inline lib/cli.js]: then https://github.com/yeoman/yo/blob/master/lib/cli.js#L66-L68 should be fixed as well

What is the deciding factor on when to keep on one line vs multiple? I broke it up like this as it was a long line.

**Comment by eddiemonge** [comment]
> Inlining `requires` is a bad idea as that won't work with ES6, so better to get into the habit of having them at the top.

In that case, all the requires should be moved.

**Comment by SBoudrias** [comment]
IMO:

- require at the top when possible (doctor was inline because at some point the require had side effect)
- multiline? If we pass 80-90 chars length line, I'd consider going multiline

File: lib/cli.js (hunk 2 above)
Comment (sindresorhus) [inline lib/cli.js]: Yes

**Pull request closed without merging**